src/main/scala/io/github/ilyazinovich/dmmf/PriceOrder.scala

```diff
@@ -22,20 +25,14 @@ object PriceOrder {
     val pricedLines: List[Either[Error, PricedOrderLine]] = orderLines.map {
```

Comment:
Do you know if there is a way to make this function even more concise?
I'm concerned about nesting map => case => action + map.

Reply:
I think it is ok. They map over different things.